### DISCUSSION TITLE
Next directus releases....

### DISCUSSION THREAD

**Discussion by koksikus** [directus/directus#15435] [Archive]
noticed that there was a change in new releases, a patch is released very quickly. This suggests that the product has finally become stable and that you are releasing patches to keep the core system functional. Will you protect these essential functions from now on to restore their functionality if you lose it? 
This is very good news because from now on you will be sure that what has worked and has been set in the system will continue to work. Personally, I am still waiting for #11058 because I consider it to be the basic functionality of the system. 
Or everything above is it all pure coincidence? 

By the way, I would like to ask how much is it possible to implement Directus to replace Jira.  People are looking for alternatives and it's good to tell them what can be achieved by implementing Directus...

One more question or idea, would it not be better to write how much it will cost to introduce a breakthrough feature and after collecting the amount, introduce it ... maybe more people will participate in financing the project

**Comment by rijkvanzanten** [comment]
> Will you protect these essential functions from now on to restore their functionality if you lose it?

Yes. Directus uses [SemVer](https://semver.org/) for versioning, so every version under 9.x.x is backwards compatible with previous releases from 9.x.x. 

> By the way, I would like to ask how much is it possible to implement Directus to replace Jira. People are looking for alternatives and it's good to tell them what can be achieved by implementing Directus...

A "ticket" is just another row in a database, so depending on what exact features of Jira you need and how you want to interact with those data points, you should be able to get very far with just Directus' included features.

> One more question or idea, would it not be better to write how much it will cost to introduce a breakthrough feature and after collecting the amount, introduce it ... maybe more people will participate in financing the project

The difficulty there is that it's really hard to tell how much a single "feature" costs. Features come in all shapes and sizes (and thus difficulty levels and timelines associated) which have different costs associated. We also have to make sure that a feature request is really well specced out and discussed before even making that estimate, as scope creep can easily ruin an estimation
